src/main/kotlin/gg/flyte/christmas/minigame/games/PaintWars.kt

```diff
@@ -169,7 +169,9 @@ class PaintWars : EventMiniGame(GameConfig.PAINT_WARS) {
         scores.entries
             .sortedByDescending { it.value }
             .take(3)
-            .also { it.forEach { formattedWinners.put(it.key, "${it.value} block${if (it.value > 1) "s" else ""}") } }
+            .also { it.forEach {
+                formattedWinners[it.key] = "${it.value} ʙʟᴏᴄᴋ${if (it.value > 1) "ѕ" else ""}" }
```

Comment:
Is this 's' here in small text? Honestly can't tell.

Reply:
Should be